client/components/Voting/voting.tsx

```diff
@@ -182,7 +183,16 @@ export default class VotingForms extends React.Component<
           didFinalvote: vote_type === 'final' ? true : this.state.didFinalvote,
           ...evt.formData,
         });
+
         alert('Submitted ' + vote_type);
+
+        if (vote_type === 'prevote' && this.state.company.company_links.prevote) {
+          window.open(
```

Comment:
Be wary about this--pop-up blockers and such. Did you test?

Reply:
Also, this means we can't re-open it if we accidentally clsoe.

Reply:
Yea this worked fine with ad block on. Not sure if the second problem is super relevant because you could always access the document again through the company page.